hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AFailureHandling.java

```diff
@@ -37,6 +40,7 @@
 import java.nio.file.AccessDeniedException;
 
 import static org.apache.hadoop.fs.contract.ContractTestUtils.*;
+import static org.apache.hadoop.fs.s3a.impl.ITestPartialRenamesDeletes.createFiles;
```

Comment:
do you think we should be moving this somewhere more reusable? S3ATestUtils, etc

Reply:
Yes that would be great.